### PR TITLE
Fix paths for docs, src

### PR DESCRIPTION
**Pull request by ArthurChen189**
Changed paths from `src/main/resources/topics-and-qrels` to `tools/topics-and-qrels`

### CODE REVIEW

File: src/test/java/io/anserini/eval/RelevanceJudgmentsTest.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public int getQrelsCount(RelevanceJudgments qrels) throws IOException{
 
   @Test
   public void testRobust04() throws IOException{
-    RelevanceJudgments qrels = new RelevanceJudgments("src/main/resources/topics-and-qrels/qrels.robust04.txt");
+    RelevanceJudgments qrels = new RelevanceJudgments("tools/topics-and-qrels/qrels.robust04.txt");
```
Comment (lintool) [inline src/test/java/io/anserini/eval/RelevanceJudgmentsTest.java:38]: Wait, how are tests passing now?

**Review by lintool — CHANGES_REQUESTED** [review]
Can we revert the changes in `src/main/python` back to the original paths? Rationale is that those are mostly one-off scripts that aren't being used anymore... I'd rather they be left alone - we can fix them if we decided to use them again. That way `git blame` will be (more) accurate.

File: src/test/java/io/anserini/eval/RelevanceJudgmentsTest.java (hunk 1 above)
Comment (ArthurChen189) [reply src/test/java/io/anserini/eval/RelevanceJudgmentsTest.java:38]: Previously, all topics were downloaded to the cache before it was searched locally.
After the fix, files are searched locally first and then download from the web if not found.

File: src/main/resources/ecir2019_axiomatic/collections.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 search_command: target/appassembler/bin/SearchCollection
-topic_root: src/main/resources/topics-and-qrels/
```
Comment (lintool) [inline src/main/resources/ecir2019_axiomatic/collections.yaml:3]: revert?

File: src/main/resources/fine_tuning/collections.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 search_command: target/appassembler/bin/SearchCollection
-topic_root: src/main/resources/topics-and-qrels/
```
Comment (lintool) [inline src/main/resources/fine_tuning/collections.yaml:3]: revert?

**Review by lintool — CHANGES_REQUESTED** [review]

**Comment by lintool** [comment]
please revert these two files?

**Review by lintool — APPROVED** [review]

**Pull request merged**